### ISSUE TITLE
Questions about pretrained models.

### ISSUE THREAD

**Issue by hello-maker** [guanjq/targetdiff#14]
Thanks for the brilliant work and sharing the code!
I have a question regarding the selection of model parameters for checkpoint: `./pretrained_models/pretrained_diffusion.pt` in the sampling.yml file under the config directory. Could you please clarify the criteria used to choose these parameters? Is it based on the checkpoint with the lowest validation loss during training? I have noticed that when using this checkpoint's model(lowest validation loss during training), the performance does not align with the results reported in the associated paper. I would appreciate any insights or guidance on potential factors that I might be overlooking in order to achieve the expected performance. 
Thank you!! 
Best regards.
